solc/CommandLineParser.h

```diff
@@ -49,6 +49,7 @@ enum class InputMode
 	StandardJson,
 	Linker,
 	Assembler,
+	AssemblyJson,
```

Comment:
Assembly is a bit ambiguous. On the CLI it usually refers to Yul. Maybe we should call it `EVMAssemblyJson` instead? Or rename `Assembler` mode to `Yul`?

Reply:
What about `ImportAssemblyJson`?

Reply:
The problem I see is just that it might not be clear that `Assembly` refers to EVM assembly rather than Yul here. On the other hand the class is named `Assembly`, so maybe it's not such a big problem...

Reply:
I just changed that now to `ImportEVMAssemblyJson`.